core/templates/pages/contributor-dashboard-page/modal-templates/translation-suggestion-review-modal.component.ts

```diff
@@ -82,34 +85,44 @@ interface ActiveContributionDict {
 })
 
 export class TranslationSuggestionReviewModalComponent implements OnInit {
-  activeContribution: ActiveContributionDict;
-  activeContributionDetails: ActiveContributionDetailsDict;
-  authorName: string;
-  activeSuggestion: ActiveSuggestionDict;
-  activeSuggestionId: string;
+  // These properties are initialized using Angular lifecycle hooks
+  // and we need to do non-null assertion. For more information, see
+  // https://github.com/oppia/oppia/wiki/Guide-on-defining-types#ts-7-1
+  activeContribution!: ActiveContributionDict;
+  activeContributionDetails!: ActiveContributionDetailsDict;
+  authorName!: string;
+  activeSuggestion!: ActiveSuggestionDict;
+  activeSuggestionId!: string;
+  contentHtml!: string | string[];
+  editedContent!: EditedContentDict;
+  errorFound: boolean = false;
+  errorMessage!: string;
```

Comment:
Move the variables without default values together above the rest.

Reply:
Done